research/src/pages/tabs.proposal.mdx

```diff
@@ -0,0 +1,283 @@
+| `start`   |  |
+| `end`   |  |
+
+_Tab_
+
+| Slot Name | Description                                                        | Fallback Content                            |
+| --------- | ------------------------------------------------------------------ | ------------------------------------------- |
+| `start`   |  |
+| `end`   |  |
+
+#### Host Classes <a href="#host-classes" id="host-classes"></a>
+
+_What classes does the component add to the host element?_
+
+| Class Name | Description |
+| ---------- | ----------- |
+
+
+#### Slotted Content Classes <a href="#slotted-content-classes" id="slotted-content-classes"></a>
+
+_What classes on slotted content does the component respond to?_
+
+| Class Name | Description |
+| ---------- | ----------- |
+
+
+#### CSS Parts <a href="#css-parts" id="css-parts"></a>
+
+_Tabs_
+
+| Part Name | Description                                                                                   |
+| --------- | --------------------------------------------------------------------------------------------- |
+| `tabs`   |  |
+| `tablist`   |  |
+| `tab`   |  |
+| `tabpanel`   |  |
+| `activeindicator`   |  |
+
+---
+
+## Behavior <a href="#behavior" id="behavior"></a>
+
+### States and Interactions <a href="#states-interactions" id="states-interactions"></a>
+
+*Tabs* can either be controlled or uncontrolled, meaning if `activeid` is passed the app author is taking control of the selected tab. When the `change` event fires it updates the `activeid` and pass a reference to the `activetab`.
+
+_List key component states, valid state transitions, and how interactions trigger state transitions._
+
+- _Are states changed as a result of keyboard? mouse? touch?_
+- _Are there corresponding keyboard interactions for mouse/touch interactions?_
+- _Are some events triggered on key down vs key press?_
+
+_Example Table_
+
+| State Group | States         | Initial State | Description                                                                                                           | Interaction/Transition                       |
+| ----------- | -------------- | ------------- | --------------------------------------------------------------------------------------------------------------------- | -------------------------------------------- |
+| `disabled`  | `true`/`false` | `false`       | When `true`, disables the control, preventing user interaction and displaying the control with a disabled appearance. | No interaction. Controlled programmatically. |
+
+### Accessibility <a href="#accessibility" id="accessibility"></a>
+
+_Consider the accessibility of the component._
+
+#### Keyboard Navigation and Focus <a href="#keyboard-navigation" id="keyboard-navigation"></a>
+
+- _Arrow vs tabbing key behavior_
+- _Modifier key effects (e.g. shift, ctrl, meta)_
+- _Does this component use focus delegation?_
+
```

Comment:
Additional questions:

- What is the behavior around disabled tabs?
- What is the behavior around tabs that move around in the UI (i.e. using flexbox order to move the active tab to be "first" in the tabgroup?

Reply:
I'm going to open these questions as issues.

Reply:
Actually I think I can inline these questions into this section as well and we can reference them in an issue.

Reply:
Added these questions, thanks!